packages/sqs/lib/utils/sqsInitter.ts

```diff
@@ -1,4 +1,6 @@
 import type { SQSClient } from '@aws-sdk/client-sqs'
+import { SetQueueAttributesCommand } from '@aws-sdk/client-sqs'
+import type { QueueAttributeName } from '@aws-sdk/client-sqs/dist-types/models/models_0'
```

Comment:
Is this import is correct? Should this include `/dist-types/models/models_0`?

Reply:
great catch! autoimport went wrong.
there were more of cases like this, fixed